state_space_dimension: 3
use_map: false
SO3_space_bounds: [-10, 10] # Low, High for x, y and z
planner_frame_start: [-2.5, 3.3, 0]  # x, y, z
planner_frame_goal: [2.5, -2.5, 0]  # x, y, z